tests/unit/backend/layers/business/test_business.py

```diff
@@ -377,13 +388,14 @@ def test_update_collection_same_doi(self):
         A collection updated with the same DOI should not trigger a Crossref call
         """
         metadata = self.sample_collection_metadata
-        links = [Link("test doi", "DOI", "http://test.doi")]
+        links = [Link("test doi", "doi", "http://test.doi")]
         metadata.links = links
 
         expected_publiser_metadata = {"authors": ["Test Author"]}
         self.crossref_provider.fetch_metadata = Mock(return_value=expected_publiser_metadata)
 
-        version = self.initialize_unpublished_collection(metadata=metadata)
+        # We need to call `business_logic.create_collection` so that the publisher metadata is populated
```

Comment:
On second thought, it would probably be better to initialize the publisher metadata similarly to the other metadata, in the test functions.